### PR TITLE
Refactor asset catalog code to use our own parsing instead of `actool`.

### PR DESCRIPTION
**Pull request by djbe**
Fixes https://github.com/SwiftGen/SwiftGen/issues/228.
Fixes https://github.com/SwiftGen/SwiftGen/issues/287.

I chose to keep the `Utils/Command.swift` code, we might want it later. Or should I remove it?

### CODE REVIEW

File: Sources/Parsers/AssetsCatalogParser.swift [hunk 1]
```diff
@@ -7,166 +7,134 @@
 import Foundation
 import PathKit
 
+public enum AssetCatalogParserError: Error, CustomStringConvertible {
```
Comment (AliSoftware) [inline Sources/Parsers/AssetsCatalogParser.swift]: Prefer a nested type inside `AsstsCatalogParser`

**Review by AliSoftware — CHANGES_REQUESTED** [review]
Except the nested error type, LGTM

**Review by AliSoftware — APPROVED** [review]

**Comment by AliSoftware** [comment]
@djbe please ensure that I didn't mess it up during my rebase (it had some conflicts) then you can merge 👍

**Comment by AliSoftware** [comment]
After thinking about it, yeah maybe we should remove `Utils/Command.swift`.

* We don't use it anymore
* If we do need to use it again maybe it won't be the exact same way, and we can still retrieve it from an old commit if really want to anyway
* Newcomers to the codebase might wonder why we have it if they don't know the history, might be confusing
* We're on the verge of a Breaking version, so it's the best time to remove unneeded stuff, later will be too late
* YAGNI

**Comment by toshi0383** [comment]
Awesome!👏

**Comment by yonaskolb** [comment]
Will this be released in SwiftGen soon? I know a forthcoming 5.0 release is in the works, but if that's a while off, maybe this could be released as a hotfix as it fixes an important bug

**Comment by AliSoftware** [comment]
As soon as I'm back from my vacation on Monday I'll do everything to fast-track the 5.0 release (which is nearly ready and just need to be pushed but I've close to no data in Rio anyway rn so will have to wait for next week for my data and time once back in Europe)

**Pull request merged**